api/v1beta1/flowcollector_types.go

```diff
@@ -404,27 +406,30 @@ type FlowCollectorFLP struct {
 	// kafkaConsumerBatchSize indicates to the broker the maximum batch size, in bytes, that the consumer will accept. Ignored when not using Kafka. Default: 10MB.
 	KafkaConsumerBatchSize int `json:"kafkaConsumerBatchSize"`
 
-	// logTypes defines the desired record types to generate. Possible values are "FLOWS" (default) to export
-	// flowLogs, "CONVERSATIONS" to generate newConnection, heartbeat, endConnection events, "ENDED_CONVERSATIONS" to generate
-	// only endConnection events or "ALL" to generate both flow logs and conversations events
+	// logTypes defines the desired record types to generate. Possible values are:<br>
+	// - `FLOWS` (default) to export regular network flows<br>
+	// - `CONVERSATIONS` to generate events for started conversations, ended conversations as well as periodic "tick" updates<br>
+	// - `ENDED_CONVERSATIONS` to generate only ended conversations events<br>
+	// - `ALL` to generate both network flows and all conversations events<br>
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:validation:Enum:="FLOWS";"CONVERSATIONS";"ENDED_CONVERSATIONS";"ALL"
 	// +kubebuilder:default:=FLOWS
 	LogTypes *string `json:"logTypes,omitempty"`
 
 	//+kubebuilder:default:="30s"
 	// +optional
-	// conversation heartbeat interval is the duration of time to wait between heartbeat reports of a conversation
+	// conversation heartbeat interval is the duration of time to wait between "tick" events of a conversation
 	ConversationHeartbeatInterval *metav1.Duration `json:"conversationHeartbeatInterval,omitempty"`
 
 	//+kubebuilder:default:="10s"
 	// +optional
-	// conversation end timeout is the duration of time to wait from the last flow log to end a conversation
+	// conversation end timeout is the time to wait after a network flow is received, to consider the conversation ended.
+	// This delay is ignored when a FIN packet is collected for TCP flows (see `conversationTerminatingTimeout` instead).
 	ConversationEndTimeout *metav1.Duration `json:"conversationEndTimeout,omitempty"`
 
 	//+kubebuilder:default:="5s"
 	// +optional
-	// conversation terminating timeout is the duration of time to wait from detected FIN flag to end a connection
+	// conversation terminating timeout is the time to wait from detected FIN flag to end a conversation. Only relevant for TCP flows.
```

Comment:
@jpinsonneau can you review in particular this part? Is this all correct and makes sense?

Reply:
Yes the frontend [translate the events as](https://github.com/netobserv/network-observability-console-plugin/blob/32dd09a3a03f3fc55c2e103ff051c84ade79cab7/web/src/utils/columns.ts#L813):
```
        switch (f.labels._RecordType) {
          case 'newConnection':
            return t('Conversation start');
          case 'heartbeat':
            return t('Conversation tick');
          case 'endConnection':
            return t('Conversation end');
          case 'flowLog':
          default:
            return t('Flow');
        }
```